dom-events/buttons.js

```diff
@@ -0,0 +1,29 @@
+
+// CLICK EVENT
+function handleClick (event) {
+  console.log('button clicked');
+  console.log(event);
+  console.log(event.target);
+  var buttonClicked = document.querySelector('.click-button');
```

Comment:
This should be done outside the function, and you want to attach the addEventListener to the variable with the reference.

Reply:
![image](https://user-images.githubusercontent.com/97073517/157358813-7932c6f1-f7b5-40ff-9796-be0cfbd55dc7.png)

![image](https://user-images.githubusercontent.com/97073517/157358844-35a5d0bd-f596-4591-819b-6e77b8496bc2.png)